scripts/distributed/find_dependencies.sh

```diff
@@ -82,11 +82,11 @@ for file in "${@}"; do
 		exit 1
 	fi;
 
-	# Get the RUNPATH.  The RUNPATH will tell us if this file was built
-	# using AT or not.
-	runpath=$(${readelf_path} -d ${file} \
-			| grep 'RUNPATH' \
-			| sed 's/^.*\: \[\(.*\)\]$/\1/')
+	# Get the interpreter.  The interpreter will tell us if this file was
+	# built using AT or not.
+	runpath=$(${readelf_path} -e ${file} \
+			| grep 'interpreter' \
+			| sed 's/\[.*\: \(.*\)\]/\1/')
```

Comment:
I tested the code before the last push, when it still used `readelf -d`, which did not show any `interpreter` in the output. I'd expect that to generate an RPM file with some wrong dependencies, but comparing the rpms and spec files between this and a recent build I had for #2008, looks like they were exactly the same.

Comparing the dependencies in the RPM files:
```
$ lammrpms=AT/valgrind-libexec/build/at15.0-0-alpha.redhat-8_ppc64le_ppc64le/rpms/ppc64le
$ valrpms=AT/valgrind-libexec/build/at15.0-0-alpha.redhat-8_ppc64le_ppc64le/rpms/ppc64le
$ for rpm in $(ls $lammrpms); do diff -s <(rpm -qp $lammrpms/${rpm}) <(rpm -qp $valrpms/${rpm}) ; done
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
Files /dev/fd/63 and /dev/fd/62 are identical
```
(notice I used temporary redirection files to compare the outputs, that's why the names with fds).

Comparing the spec files:
```
$ diff -s AT/{lamm-runpath-pr,valgrind-libexec}/build/tmp.at15.0-0-alpha.redhat-8_ppc64le_ppc64le/rpmbuild/SPECS/advance-toolchain.spec 
Files AT/lamm-runpath-pr/build/tmp.at15.0-0-alpha.redhat-8_ppc64le_ppc64le/rpmbuild/SPECS/advance-toolchain.spec and AT/valgrind-libexec/build/tmp.at15.0-0-alpha.redhat-8_ppc64le_ppc64le/rpmbuild/SPECS/advance-toolchain.spec are identical
$ diff -s AT/{lamm-runpath-pr,valgrind-libexec}/build/tmp.at15.0-0-alpha.redhat-8_ppc64le_ppc64le/rpmbuild/SPECS/advance-toolchain_at-compat.spec
79,80c79,80
< mkdir -p ${RPM_BUILD_ROOT}/home/mscastanho/AT/lamm-runpath-pr/opt/at14.0/lib \
<          ${RPM_BUILD_ROOT}/home/mscastanho/AT/lamm-runpath-pr/opt/at14.0/lib64
---
> mkdir -p ${RPM_BUILD_ROOT}/home/mscastanho/AT/valgrind-libexec/opt/at14.0/lib \
>          ${RPM_BUILD_ROOT}/home/mscastanho/AT/valgrind-libexec/opt/at14.0/lib64
102,103c102,103
< /home/mscastanho/AT/lamm-runpath-pr/opt/at14.0/lib
< /home/mscastanho/AT/lamm-runpath-pr/opt/at14.0/lib64
---
> /home/mscastanho/AT/valgrind-libexec/opt/at14.0/lib
> /home/mscastanho/AT/valgrind-libexec/opt/at14.0/lib64
```
(the only difference here is the destination path, which is expected).

So I guess this is working?

But I suggest renaming the `runpath` variable to `interpreter` on lines 87, 90 and 91. It can cause confusion in the future since runpath is not used anymore.